Highway.h

```diff
@@ -1,8 +1,11 @@
 #pragma once
-
+#include "Vehicle.h"
```

Comment:
If you're not using `Vehicle` at all in this header file, don't include it
forward declare it
include it in the cpp file.
- [ ] revise